index.html

```diff
@@ -5,63 +5,259 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
```

Comment:
- [ ] Please make sure your website has a good page title. Take a look at [this article](https://www.w3.org/WAI/test-evaluate/preliminary/#titlehttps://www.w3.org/WAI/test-evaluate/preliminary/#title) for more information about it.
- It adequately and briefly describes the content of the page. For example:
 - Poor titles:
   - Microverse project | Your portfolio.
   - Portifolio project
 - Better page titles:
   - Mark's portfolio
   - Mark full-stack webdeveloper